socialNetwork/src/UserTimelineService/UserTimelineHandler.h

```diff
@@ -297,6 +326,11 @@ void UserTimelineHandler::ReadUserTimeline(
         _redis_client_pool->zadd(std::to_string(user_id),
                                redis_update_map.begin(),
                                redis_update_map.end());
+      else if (_redis_replica_pool && _redis_primary_pool) {
```

Comment:
check `_redis_primary_pool` only, no need to check replica pool if it's not used